Source/WebCore/rendering/MarkedText.h

```diff
@@ -68,17 +68,33 @@ struct MarkedText {
         Frontmost
     };
 
-    unsigned startOffset;
-    unsigned endOffset;
-    Type type;
+    MarkedText(unsigned startOffset, unsigned endOffset, Type type, const RenderedDocumentMarker* marker = nullptr, AtomString highlightName = { }, int priority = 0) : startOffset(startOffset), endOffset(endOffset), type(type), marker(marker), highlightName(highlightName), priority(priority) { };
+    MarkedText(WTF::HashTableDeletedValueType) : m_isHashTableDeletedValue(true) { };
+    MarkedText() { };
+
+    unsigned startOffset { 0 };
+    unsigned endOffset { 0 };
+    Type type { Unmarked };
     const RenderedDocumentMarker* marker { nullptr };
     AtomString highlightName { };
-
+    int priority { 0 };
+    bool m_isHashTableDeletedValue { false };
```

Comment:
this should probably be private

Reply:
Did you mean priority should be private or the bool? Or both?

Reply:
just `m_isHashTableDeletedValue`, since clients of this struct really shouldn't know or care about it